### PR TITLE
Reformatted hierarchy

### PR DESCRIPTION
**Pull request by jonslaught**
@lpappone your PR looked good to me except the organization, so I made this PR to group things a bit differently. @tylerbrandt fyi

### CODE REVIEW

File: src/content/javascript/personalization/index.md [hunk 1]
```diff
@@ -377,10 +377,161 @@ utils.waitUntil(function() {
...
+
+##### *Parameters*
+
+- `type`: "addListener"
+- `filter` (object): The events to listen for
+  - `type` (string)
+  - `name` (string)
+- `handler` (function): A callback to run when an event fires. The function takes a single `data` object that varies based on the event.
+
+##### *Examples*
```
Comment (lpappone) [inline src/content/javascript/personalization/index.md]: In other places, you changed 'example' to 'usage'.  I think this should be internally consistent. Is there a reason you changed it?

File: src/content/javascript/personalization/index.md [hunk 2]
```diff
@@ -377,10 +377,161 @@ utils.waitUntil(function() {
...
+  }
+);
+
+// Specify a content-enabled attribute by name.
+dcp.waitForAttributeValue({datasourceId: 123, attributeName: 'Preferred Locale'}).then(
+  function (attributeValue) {
+  }
+);
+```
+
```
Comment (lpappone) [inline src/content/javascript/personalization/index.md]: Kind of hard to tell what you did here without going through line by line - is all the above just moved from below, or are there changes to the content?

**Comment by lpappone** [comment]
Thanks @jonslaught I'm fine with this, except the questions added inline.

File: src/content/javascript/personalization/index.md (hunk 1 above)
Comment (jonslaught) [inline src/content/javascript/personalization/index.md]: I think a good pattern is to use "examples" when we actually have example data/parameters being filled in, like the listener in this case that shows console.logging. Usage makes sense when we're just showing how you would run the campaign, without any example data.

File: src/content/javascript/personalization/index.md (hunk 2 above)
Comment (jonslaught) [inline src/content/javascript/personalization/index.md]: Yep, just moved it, no other changes besides Examples --> Usage in a few places

**Comment by jonslaught** [comment]
@lpappone responded to your comments. Do you want to merge this into your branch and then we can land from there into master?

File: src/content/javascript/personalization/index.md (hunk 1 above)
Comment (tylerbrandt) [inline src/content/javascript/personalization/index.md]: > I think a good pattern is to use "examples" when we actually have example data/parameters being filled in, like the listener in this case that shows console.logging. 

Ok, as long as it's consistent.

> Usage makes sense when we're just showing how you would run the campaign, without any example data.

I think that distinction just confuses matters. Some methods don't have parameters. Are they then always "Usage", never "Examples"? I would rather just pick one and stick with it for all code snippets.

**Comment by jonslaught** [comment]
Ok, @tylerbrandt @lpappone I changed all examples headings to usage

**Comment by tylerbrandt** [comment]
Why can't we just leave it as "Examples"; all the existing content uses that...

**Comment by jonslaught** [comment]
Because most of these aren't examples. Examples would have example response
data.

**Pull request merged**